### PR TITLE
fix

### PR DESCRIPTION
**Pull request by uk4z**

### CODE REVIEW

File: src/lib/index.ts [hunk 1]
```diff
@@ -22,7 +22,7 @@ export function displayUsFormat(value: BigNumber, decimals = 2): string {
 	const converted = truncatedValue.toFixed(decimals);
 
 	return new Intl.NumberFormat('en-US', {
-		minimumFractionDigits: decimals,
+		minimumFractionDigits: 0,
```
Comment (leo-ship-it) [inline src/lib/index.ts:25]: ?

File: src/routes/stake/Stake.svelte [hunk 2]
```diff
@@ -133,11 +133,8 @@
 		<div class="reward">
 			<p style:margin-right={'2.5em'}>
 				Future WTN Airdrop:
-				{#if exchangeRate && totalIcpDeposited}
-					{computeRewards(
-						totalIcpDeposited,
-						computeReceiveAmount(true, BigNumber($inputAmount), exchangeRate)
-					).toFixed(2)}
+				{#if totalIcpDeposited && !BigNumber($inputAmount).isNaN()}
+					{displayUsFormat(computeRewards(totalIcpDeposited, BigNumber($inputAmount)), 8)}
```
Comment (leo-ship-it) [inline src/routes/stake/Stake.svelte:137]: 8 decimals for a number that can have 3 decimals at most?

File: src/lib/index.ts (hunk 1 above)
Comment (uk4z) [reply src/lib/index.ts:25]: 15.000000000 becomes 15

File: src/routes/stake/Stake.svelte (hunk 2 above)
Comment (uk4z) [reply src/routes/stake/Stake.svelte]: not necessary that's true however it will not display 8 decimals for nothing but just up to 8 decimals.

**Pull request merged**